whipper/test/test_common_mbngs.py

```diff
@@ -44,6 +44,27 @@ def testTrackTitle(self):
         track1 = metadata.tracks[0]
         self.assertEqual(track1.title, 'Brownsville Turnaround')
 
+    def testClassicalTrack(self):
+        """
+        Test if composers and performers are extracted properly
+        see https://github.com/whipper-team/whipper/issues/191
+        """
+        # Using: Mama Said - Lenny Kravitz
+        # https://musicbrainz.org/release/410f99f8-a876-3416-bd8e-42233a00a477
+        filename = 'whipper.release.410f99f8-a876-3416-bd8e-42233a00a477.json'
+        path = os.path.join(os.path.dirname(__file__), filename)
+        with open(path, "rb") as handle:
+            response = json.loads(handle.read().decode('utf-8'))
+
+        metadata = mbngs._getMetadata(response['release'],
+                                      discid='bIOeHwHT0aZJiENIYjAmoNxCPuA-')
+        track1 = metadata.tracks[0]
+        self.assertEqual(track1.composers, 'Hal Fredericks and Michael Kamen')
+        self.assertEqual(track1.performers, ['bass guitar, guitar family, '
+                                             'mellotron, membranophone: Lenny '
+                                             'Kravitz', 'guest guitar: Slash',
```

Comment:
```suggestion
                                             'Kravitz',
                                             'guest guitar: Slash',
```
(This makes the list more (human) readable.)